src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -132,14 +132,9 @@
     @if context.messages.filterBundlingMethods(ifaceConfig, context.getSupportedMethods(service))
 
       BUNDLE_DESCRIPTORS = {
-        @let ifaceConfig = context.getApiConfig.getInterfaceConfig(service)
-          @join method : context.messages.filterBundlingMethods(ifaceConfig, context.getSupportedMethods(service)) on {@", "}.add(BREAK)
-            @let bundling = ifaceConfig.getMethodConfig(method).getBundling(), \
-                 methodName = context.upperCamelToLowerUnderscore(method.getSimpleName)
-              "{@methodName}" => Google::Gax::BundleDescriptor.new(
-                {@bundleDescriptorBody(bundling, method)})
-            @end
-          @end
+        @join descriptor : context.getBundlingDescriptors(service) on {@", "}.add(BREAK)
```

Comment:
I'm assuming that the baseline doesn't change because your new form here exactly replicates the original output?